.github/workflows/docker.yml

```diff
@@ -121,7 +121,7 @@ jobs:
           touch "/tmp/digests/${digest#sha256:}"
 
       - name: Upload digest
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: digests-${{ inputs.registry }}-${{ inputs.image_name }}-${{ steps.tag.outputs.tags }}
```

Comment:
This doesn't work, each artifact needs a unique name in v4. This upload to the same name for every matrix entry.

Reply:
(slashes aren't valid either, need to fix that after sleeping :))